syft/sandbox.py

```diff
@@ -1,11 +1,19 @@
 import importlib
+import numpy as np
 
 from syft.frameworks.torch.hook.hook import TorchHook
 from syft.workers.virtual import VirtualWorker
 from syft.grid.private_grid import PrivateGridNetwork
 
 from syft.exceptions import DependencyError
 
+try:
+    from tensorflow.keras import datasets
+
+    tf_datasets_available = True
```

Comment:
MAYBE: Why not use ```dependecy_check.tensorflow_available```?

Reply:
it's not the same version of TF
tensorflow_available is FALSE if TF<2
while the sandbox should be usable

Reply:
Ahhhh...:(

Reply:
Then it looks ok :sunglasses: